classes/test.php

```diff
@@ -335,6 +334,59 @@ public function addUnsupportedMethod($testMethod, $reason)
         return $this;
     }
 
+    /**
+     * The `setUp` method in atoum maps to the `setUpBeforeClass` method in
+     * PHPUnit.
+     */
+    protected function callSetUp()
+    {
+        static::setUpBeforeClass();
+    }
+
+    /**
+     * The `tearDown` method in atoum maps to the `tearDownAfterClass` method
+     * in PHPUnit.
+     */
+    protected function callTearDown()
+    {
+        static::tearDownAfterClass();
+    }
+
+    /**
+     * The `beforeTestMethod` method in atoum maps to the `setUp` method in
+     * PHPUnit.
+     */
+    protected function callBeforeTestMethod($testMethod)
+    {
+        $this->beforeTestMethod($testMethod);
```

Comment:
why not using `parent:: callBeforeTestMethod($testMethod);` here ?

Reply:
Because I don't want to inherit the behavior of `atoum\test`.

I changed my mind several times on this one. But at the end, I decided that the behaviors are so much different, I don't want a change on `atoum\test` to impact this extension.